docs/topics/db/multi-db.txt

```diff
@@ -159,9 +159,10 @@ A database Router is a class that provides up to four methods:
 
     Return ``True`` if a relation between ``obj1`` and ``obj2`` should be
     allowed, ``False`` if the relation should be prevented, or ``None`` if
-    the router has no opinion. This is purely a validation operation,
-    used by foreign key and many to many operations to determine if a
-    relation should be allowed between two objects.
+    the router has no opinion. If no router has an opinion, the default
```

Comment:
This documentation change isn't related to the bug fix, correct? Please submit a separate pull request.